src/main/java/model/figure/Circle.java

```diff
@@ -0,0 +1,23 @@
+package model.figure;
+
+import model.Figure;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public int calculateArea() {
+        return (int) (Math.PI * (radius * radius));
+    }
+
+    @Override
+    public void print() {
+        System.out.println("Figure: Circle, area:  " + calculateArea()
```

Comment:
The method 'print' is not conceptually similar to 'calculateArea'. According to the checklist, different behaviors should not be put into a single interface or class. Consider separating the responsibilities into different interfaces or classes.